book/src/setup.md

```diff
@@ -38,22 +38,29 @@ settings. All settings configured via the USB interface are only applied when St
 up. Any modifications that occur after boot over the MQTT interface will occur after the initial
 settings configured via USB.
 
+> **Note:** Settings configured via USB do not take immediat effect. Instead, they will apply after
+> the device is rebooted.
+
+> **Note:** Settings configured via USB are only initial settings. Any modifications
+> that occur after startup (i.e. via MQTT) will take precedent over those configured via USB. MQTT
```

Comment:
We should make sure that in the docs we don't flip-flop between saying "settings configured via USB" and "settings stored in Flash" as both are the same.

Reply:
I don't think "settings stored in flash" is mentioned anywhere in the docs. Did you have a specific location in mind?

Reply:
You are right. Looks good!